predicators/approaches/grammar_search_invention_approach.py

```diff
@@ -442,6 +501,66 @@ def _get_feature_ranges(
         return feature_ranges
 
 
+@dataclass(frozen=True, eq=False, repr=False)
+class _FeatureDiffInequalitiesPredicateGrammar(
+        _SingleFeatureInequalitiesPredicateGrammar):
+    """Generates features of the form "|0.feature - 1.feature| >= c" or
+    "|0.feature - 1.feature| <= c". This is also capable of
```

Comment:
update docstring

Reply:
in addition to removing the single-feature comment, isn't it also the case that we're only generating `|0.feature - 1.feature| <= c` and not `|0.feature - 1.feature| >= c`? (which is good, because we can basically get the latter by applying negation later)